webapp/src/components/modals/channel_subscriptions/select_channel_subscription.tsx

```diff
@@ -112,9 +136,48 @@ export default class SelectChannelSubscriptionInternal extends React.PureCompone
         );
     }
 
+    renderSubscriptionTemplateRow(sub: ChannelSubscription, projectName: string, showInstanceColumn: boolean, instanceName: string): JSX.Element {
+        return (
+            <tr
+                key={sub.id}
+                className='select-channel-subscriptions-row'
+            >
+                <td>
+                    <span>{sub.name || '(no name)'}</span>
+                </td>
+                <td>
+                    <span>{projectName}</span>
+                </td>
+                {showInstanceColumn && (
+                    <td>
+                        <span>{instanceName}</span>
+                    </td>
+                )}
+
+                <td>
+                    <button
+                        className='style--none color--link'
+                        onClick={(): void => this.props.showEditSubscriptionTemplate(sub)}
```

Comment:
```suggestion
                        onClick={(): void => this.props.showEditSubscriptionTemplate(sub)}
```
I don't think we need to define this `void` return type